dev/integration_tests/windows_startup_test/windows/flutter/generated_plugins.cmake

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
This one gets checked in though!

Reply:
Removed the other file but kept this one!